build-aux/summit.cmake

```diff
@@ -39,5 +44,5 @@ set(MAGMA_DIR $ENV{OLCF_MAGMA_ROOT} CACHE PATH
   "Path to the MAGMA installation directory. Hint for CMake to find MAGMA.")
 
 # FFTW paths.
-set(FFTW_INCLUDE_DIR $ENV{OLCF_FFTW_ROOT}/include CACHE PATH "Path to fftw3.h.")
-set(FFTW_LIBRARY $ENV{OLCF_FFTW_ROOT}/lib/libfftw3.so CACHE FILEPATH "The FFTW3(-compatible) library.")
+set(FFTW_INCLUDE_DIRS $ENV{OLCF_FFTW_ROOT}/include CACHE PATH "Path to fftw3.h.")
+set(FFTW_LIBRARIES $ENV{OLCF_FFTW_ROOT}/lib/libfftw3.so CACHE FILEPATH "The FFTW3(-compatible) library.")
```

Comment:
I think you should set the HPX directory in this file, like the other libraries.

Reply:
Thanks for notice. I just updated HPX dir in this file.